packages/contracts-bedrock/scripts/fpac/Makefile

```diff
@@ -0,0 +1,42 @@
+# ENV
+monorepo_root := "../../../.."
+deploy_config_path := "../../deploy-config"
+tmp := $(shell mktemp)
+
+# Help menu
+.PHONY: help
+help:
+	@echo "Available commands:"
+	@echo "  make cannon-prestate chain=<chain>"
+	@echo "  make deploy-fresh chain=<chain> args=<args>"
+	@echo "  make upgrade-game-impl chain=<chain> dgf=<dgf> vm=<vm> args=<args>"
+	@echo "  make update-init-bond chain=<chain> dgf=<dgf> game-type=<game-type> new-init-bond=<new-init-bond> args=<args>"
```

Comment:
Here's a more less-redundant, more maintainable way of creating a help menu for a Makefile: https://dwmkerr.com/makefile-help-command/.

Not a blocker but just wanted to share since it saves us the headache of needing to update this whenever arguments or syntax changes for the targets.

Reply:
This is sick, done